ImageFeed/ImagesList/ImagesListViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  ViewController.swift
+//  ImageFeed
+//
+//  Created by Тимофей Спирин on 09.12.2024.
+//
+
+import UIKit
+
+class ImagesListViewController: UIViewController {
+    
+    @IBOutlet private var table: UITableView!
+    
+    private let photosName: [String] = Array (0..<20).map{ "\($0)" }
+    
+    private lazy var dateFormatter: DateFormatter = {
```

Comment:
Если нужен русский формат даты, можно задать dateFormat и locale, например так:

    private lazy var dateFormatter: DateFormatter = {
        let formatter = DateFormatter()
        formatter.dateFormat = "d MMMM yyyy"
        formatter.locale = Locale(identifier: "ru_RU")
        return formatter
    }()
    
    Но если важен просто длинный формат даты без точного контроля за отображением, можно использовать dateStyle = .long, как в твоём варианте.
    
```suggestion
    private lazy var dateFormatter: DateFormatter = {
          let formatter = DateFormatter()
        formatter.dateStyle = .long
        formatter.timeStyle = .none
        return formatter
    }()
```